docs/ota-client-guide/modules/ROOT/pages/intro-prep.adoc

```diff
@@ -16,7 +16,7 @@ Here are our recommended steps for integration:
 
 * xref:add-environments.adoc[*Set up different environments*]
 +
-Avoid mixing up test software and production software by uploading the software under different environments.
+To avoid mixing up test software and production software upload the software to different environments.
```

Comment:
```suggestion
To avoid mixing up test software and production software, upload the software to different environments.
```

Reply:
The comma is still missing :)

Reply:
I think it wasn't saved in my commit. I'll try it again.